src/ducks/Events.js

```diff
@@ -159,12 +171,21 @@ export const editEvent = (event) => {
  */
 
 export const deleteEvent = (event) => {
-	firebase.database().ref("events").update({ [event.id]: null })
-		.then(() => {
-			firebase.database().ref(`committees/${event.committee}/events/`).update({ [event.id]: null });
-		})
-		.then(() => Alert.alert("Event deleted", { type: "success", title: "Successful" }))
-		.catch(() => Alert.alert("Event deletion failed", { type: "error", title: "Failure" }));
+	let eventRef = firebase.database().ref(`events/${event.id}`);
+
+	return (dispatch) => {
```

Comment:
Is there a reason this needs to be dispatch to the reducer?

Reply:
If there is then this function needs to be moved to the section that does use redux. Of this ducks file